pkg/v1/tkg/test/tkgctl/shared/package.go

```diff
@@ -267,7 +267,7 @@ func verifyClusterBootstrap(ctx context.Context, c client.Client, clusterBootstr
 		expectedClusterBootstrap.Spec.CNI.ValuesFrom.ProviderRef = &corev1.TypedLocalObjectReference{
 			APIGroup: &apiGroup,
 			Kind:     "CalicoConfig",
-			Name:     GeneratePackageSecretName(clusterBootstrap.Name, "calico"),
```

Comment:
for clusters made with custom CB, the name of the config object is the same as the CB. But for those CB created from CBT, the above name is used. So both these names can be valid for different scenarios. 
Now why this change is needed?

Reply:
Thanks for the review! As Shyaam mentioned in the channel, this change is in the path of Calico CNI. And since Calico is the case of custom CB, I think it is safe to change here. The CB created from CBT won't go in this path as the CNI won't be Calico.

Reply:
Ok, so just noticed ```strings.Contains(clusterBootstrap.Spec.CNI.RefName, "calico")``` check, and calico always use custom CB. We should be good then. Thanks for the clarification.